opm/parser/eclipse/EclipseState/Schedule/TimeMap.hpp

```diff
@@ -83,17 +83,17 @@ namespace Opm {
             }
         };
 
-        std::vector<std::time_t> m_timeList;
-
         bool isTimestepInFreqSequence (size_t timestep, size_t start_timestep, size_t frequency, bool years) const;
         size_t closest(const std::vector<size_t> & vec, size_t value) const;
         void addTStep(int64_t step);
         void addTime(std::time_t newTime);
-        void addFromDATESKeyword( const DeckKeyword& DATESKeyword );
```

Comment:
Just out of curiosity, why do you remove `addFromDATES()`?  Is there a reason that pasting its code inline is preferable?

Reply:
The point is that there is logic - looking for the restart date - for each *record* in the keyword, previously the keyword was read in it's entirety.

Reply:
> The point is that there is logic - looking for the restart date - for each _record_ in the keyword,

Right.

> previously the keyword was read in it's entirety.

Could you not update the function to also do that?

Reply:
Yes - the function could have been updated; but then some temporary state would be passed between the updated `addFromDATESKeyword()` and the main loop - I considered that to be more ugly than the current solution.